README.md

```diff
@@ -1,27 +1,43 @@
 kubernetes-mesos
 ================
 
-When [Google Kubernetes][2] meets [Apache Mesos][3]
+[Google Kubernetes](https://github.com/GoogleCloudPlatform/kubernetes) on [Apache Mesos](http://mesos.apache.org/)
 
-Kubernetes and Mesos are a match made in heaven.
-Kubernetes enables the Pod, an abstraction that represents a group of co-located containers, along with Labels for service discovery, load-balancing, and replication control.
-Mesos provides the fine-grained resource allocations for pods across nodes in a cluster, and facilitates resource sharing among Kubernetes and other frameworks running on the same cluster.
+Disclaimer: Kubernetes is pre-release quality, and not yet recommended for production systems.
```

Comment:
can we say "Kubernetes-Mesos" here instead? Let the k8s team make their own announcements w/ respect to production fitness